src/language/pythonLanguage.ts

```diff
@@ -1,30 +1,67 @@
 import { StringOrRegExp } from '@cucumber/cucumber-expressions'
+import { RegExps } from '@cucumber/cucumber-expressions/dist/cjs/src/ParameterType'
 
-import { Language, TreeSitterSyntaxNode } from './types.js'
+import { childrenToString, filter, NO_QUOTES } from './helpers.js'
```

Comment:
Fix imports. You can run `npm run eslint` to list linting problems and `npm run eslint-fix` to auto-fix some of them.